examples/docker-simple-on-ubuntu/main.tf

```diff
@@ -0,0 +1,197 @@
+variable "ubuntu_os_version_allowedvalues" {
+  type = "map"
+
+  default = {
+    "16.04.0-LTS" = "16.04.0-LTS"
+    "16.10"       = "16.10"
+  }
+}
+
+variable "config" {
+  type = "map"
+
+  default = {
+    "namespace"                     = "docker"
+    "vm_size"                       = "Standard_F1"
+    "vm_image_publisher"            = "Canonical"
+    "vm_image_offer"                = "UbuntuServer"
+    "storage_account_type"          = "Standard_LRS"
+    "network_address_prefix"        = "10.0.0.0/16"
+    "network_subnet_prefix"         = "10.0.0.0/24"
+    "network_public_ipaddress_type" = "Static"
```

Comment:
_same as above_ - could we split these out into variables if we're expecting the user to assign them? in an example I'd expect the defaults to be suitable in most cases?